src/App.js

```diff
@@ -449,6 +436,20 @@ class App extends Component {
                         <option value="Iota"> Iota (HD) </option>
                         <option value="Zeta"> Zeta (Series) </option>
                     </select>
+                    <select
+                        type="text"
+                        ref={this.searchInput}
+                        className="year-selector"
+                        id='mySelect'
+                        autoFocus={true}
+                        onChange={this.setYear}
+                        >
+                          <option value = 'default' className='op' >Default</option>
```

Comment:
Good day @Amenacliff, Lastly before we merge your PR, can you please change the **default** `option` name to **All year**

Reply:
okay sure

Reply:
> okay sure

**All Year** not **All Years**

Reply:
Tried switching to another engine, this  cleared the default and rendered an empty option

please, add the `selected="selected"` attribute to the default year option